enhancements/oc/day2-add-nodes.md

```diff
@@ -0,0 +1,264 @@
+installer. The user generates an ISO image via an `oc adm` command with an optional configuration file to describe the new nodes. The image 
+is used to boot the hosts so that they will be added to the cluster without any further action, except for the final approval that remains a 
+manual operation up to the user. An additional `oc adm` command will allow the user to monitor the progress of the task.
+
+## Motivation
+The task of adding a node to an already existing non-IPI cluster (regardless of the installation approach adopted) still remains a complex 
+task for the end user, with many different methods available that in some cases may lead to custom automations difficult to maintain. In 
+general, scaling up an on-premises cluster should be a simple operation, mostly automated, for the convenience of the end user.
+
+Currently the agent-based installer method significantly improved the user experience for deploying a cluster - especially in disconnected 
+scenarios - by focusing on full automation of the OpenShift-specific parts of the process, and leaving to the user the decision to automate 
+or not the required cluster infrastructure. The approach addressed several pain points, in particular by not requiring any additional provisioning 
+hardware, and by applying a number of live pre-flight validations in the target environment to spot in advance those issues that could hamper 
+the setup. Also, capturing all the necessary elements for the installation in a single ISO image simplified the application and distribution 
+into different environments. 
+
+So the current enhancement proposes a standard and unified way for adding nodes to an on-premises cluster, by replicating the same approach 
+successfully adopted for the installation process, so that the end user could benefit from a more consistent and simple workflow. In addition, 
+the standardization will make obsolete the hand-rolled UPI method, the need to use MCE to manage local clusters, and (unsupported) field-developed 
+automations that are expensive to maintain.
+
+### User Stories
+- As a cluster administrator, I want to add (or replace) one or more nodes to an existing cluster (potentially deployed in a disconnected environment) 
+  using the same simplified agent-based workflow already adopted for the installation
+- As a cluster administrator, I want to provide the initial static network configuration required for each different host that will be added to the cluster
+- As a cluster administrator, I want to validate the nodes to be added before initiating the procedure and be informed of any validation failures
+- As a cluster administrator, I want to be able to monitor the process of adding new nodes
+
+### Goals
+- Allow the user to easily add one or more worker nodes to an existing (disconnected) non-IPI cluster, with the minimum amount of configuration requested
```

Comment:
I'd say to any cluster, regardless of the installation method. While the focus is on non-IPI clusters, IPI clusters aren't discarded from this workflow (as you point out later with the use case where the user can't use BMCs in bare metal clusters) and we expect users to be able to use it (and they will use it for different reasons, no BMCs is one). So I'd reword this point to be less excluding of IPI clusters.

Reply:
Maybe we say clusters where MachineSet scaleout is not available/possible. That's a more precise definition of what we're targeting.

Reply:
That sounds reasonable to me (see previous comment also)